### PR TITLE
upgrade to cassandra 3.1

### PR DESCRIPTION
**Pull request by MassimoCappellano**
Hi Rajat,

I've done this upgrade to version 3.1 of cassandra, and exposed another port, `OPENSHIFT_CASSANDRA_NATIVE_TRANSPORT_PORT`, port for connecting with client, on NATIVE TRANSPORT protocol.

Massimo

### CODE REVIEW

**Comment by ravilution** [comment]
Hi, can you review and pull this. I am in need of this.

**Comment by MassimoCappellano** [comment]
Hi, I have yet done request to do pull

**Comment by MassimoCappellano** [comment]
I have reviewed and seems ok

File: metadata/manifest.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ Categories:
   - database
   - embedded
 Website: http://cassandra.apache.org
-Source-Url: https://github.com/rajatchopra/openshift-origin-cartridge-cassandra.git
+Source-Url: https://github.com/MassimoCappellano/openshift-origin-cartridge-cassandra.git
```
Comment (rajatchopra) [inline metadata/manifest.yml]: This is my only concern.. this repo should likely not point to another repo

**Comment by MassimoCappellano** [comment]
I hope now is all ok.

**Pull request merged**